recipes/py-plumed/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+cd python
+make pip
```

Comment:
Can we use multiple jobs here?
```suggestion
make -j${CPU_COUNT} pip
```

Reply:
This target is just copying two files from another directory, so I think it does not make sense to parallelize it:
````
pip:
        cp ../src/wrapper/Plumed.h .
        cp ../VERSION .
````